Application/EdFi.Ods.Api.NetCore/Startup/OdsStartupBase.cs

```diff
@@ -48,17 +48,17 @@ public abstract class OdsStartupBase
         private const string CorsPolicyName = "_development_";
         private readonly ILog _logger = LogManager.GetLogger(typeof(OdsStartupBase));
 
-        public OdsStartupBase(IWebHostEnvironment env)
+        public OdsStartupBase(IWebHostEnvironment env, IConfiguration configuration)
         {
             _logger.Debug("Loading configuration files");
```

Comment:
this debug statement would be moved to the program.cs file

Reply:
Done